Commonvars.mk

```diff
@@ -54,7 +54,8 @@ EXTRA_TAG ?= -dev
 PLUGIN_TAG := $(VERSION_TAG)$(EXTRA_TAG)
 
 # plugin name - used as a base for full plugin name and container for extracting rootfs
-PLUGIN_NAME=$(DOCKER_HUB_REPO)/docker-volume-vsphere
+PLUGIN_NAME ?= $(DOCKER_HUB_REPO)/docker-volume-vsphere
```

Comment:
curious to know here .. `?=` why is this change? 

1. is it passed or set?
2. don't we need for L58?